TabloidMVC/Models/Post.cs

```diff
@@ -29,6 +30,7 @@ public class Post
         [DisplayName("Category")]
         public int CategoryId { get; set; }
         public Category Category { get; set; }
+        public List<Category> CategoryOptions { get; set; }
```

Comment:
We can't put a list of category options directly on the Post model, because there's no relationship in the db to a post and _all_ the categories-- just the one that get selected. This is a good case for a view model!